cmd/minishift/cmd/config/view.go

```diff
@@ -74,7 +75,9 @@ func configView() error {
 		if excluded {
 			continue
 		}
-
+		if configViewFormat != DefaultConfigViewFormat {
+			configViewFormat = fmt.Sprintln(configViewFormat)
```

Comment:
We need this because what was happening is variable value coming from shell was consumed directly as it is.